### ISSUE TITLE
Sample rate (fs) - No default. Must select either 8000 or 16000.  

### ISSUE THREAD

**Issue by risqaliyevds** [yxlu-0102/MP-SENet#20]
I am training with 48k but in validation I get this 

        Run model on reference(ref) and degraded(deg)
        Sample rate (fs) - No default. Must select either 8000 or 16000.
        Note there is narrow band (nb) mode only when sampling rate is 8000Hz.
        
        
How to fix it or it is imposible

**Comment by yxlu-0102** [comment]
Could you let me know when you encountered this issue, and which code corresponds to this error? 
Did it occur during the calculation of PESQ?

**Comment by risqaliyevds** [comment]
{
    "num_gpus": 8,
    "batch_size": 16,
    "learning_rate": 0.0005,
    "adam_b1": 0.8,
    "adam_b2": 0.99,
    "lr_decay": 0.99,
    "seed": 1234,
    
    "dense_channel": 64,
    "compress_factor": 0.3, 
    "num_tsconformers": 4,
    "beta": 2.0,

    "sampling_rate": 48000,
    "segment_size": 96000,
    "n_fft": 800,
    "hop_size": 300, 
    "win_size": 800, 

    "num_workers": 64,

    "dist_config": {
        "dist_backend": "nccl",
        "dist_url": "tcp://localhost:54321",
        "world_size": 1
    }
}


####
I set this setttings.
I couldn't find this error where located that is why I'm asking.

**Comment by yxlu-0102** [comment]
I understand that the metric discriminator needs to compute PESQ during training, but PESQ only supports speech below 16kHz.

**Comment by yxlu-0102** [comment]
You can remove the metric discriminator, or find another metric that can be calculated under a 48 kHz sampling rate, such as [ViSQOL](https://github.com/google/visqol).

**Comment by risqaliyevds** [comment]
aha thanks
